data/presets/power/catenary_mast.json

```diff
@@ -0,0 +1,28 @@
+{
+    "icon": "temaki-power_pole",
+    "fields": [
+        "ref",
+        "operator"
+    ],
+    "moreFields": [
+        "height",
+        "line_attachment",
+        "line_management",
+        "manufacturer",
+        "material"
+    ],
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "terms": [
+        "overhead line"
+    ],
+    "tags": {
+        "power": "catenary_mast"
+    },
+    "name": "Catenary Mast",
```

Comment:
The [most common](https://books.google.com/ngrams/graph?content=catenary+mast%2Ccatenary+pole&year_start=1800&year_end=2019&corpus=en-2019&smoothing=3) term would be “catenary pole”, with “catenary mast” being [nearly nonexistent](https://books.google.com/ngrams/graph?content=catenary+mast%2Ccatenary+pole&year_start=1800&year_end=2019&corpus=en-US-2019&smoothing=3) in American English. This is the case even if the structure includes a mast arm (compare “light pole”). (The spelling `catenary_mast` was [reportedly](https://wiki.openstreetmap.org/wiki/Special:Diff/1252192) coined by a couple of German railway mappers without wider discussion.)